km/km_filesys_private.h

```diff
@@ -60,7 +60,9 @@ typedef enum km_file_how {
    KM_FILE_HOW_SOCKETPAIR1 = 7,
    KM_FILE_HOW_RECVMSG = 8,
    KM_FILE_HOW_EVENTFD = 9, /* eventfd() */
-   KM_FILE_HOW_TIMERFD = 10
+   KM_FILE_HOW_TIMERFD = 10,
+   KM_FILE_HOW_WATCH = 11,
+   KM_FILE_HOW_WATCH1 = 12
```

Comment:
Would suggest you add code to km_fs_core_notes_write() to disallow snapshots of payloads that use inotify until we can support snapshoting inotify fd's.